app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -190,3 +190,54 @@ input.search-bar {
 .form-control:focus {
   box-shadow: none;
 }
+
+
+//Semantic UI pagination
+.table-pagination-wrapper {
```

Comment:
Since this has nothing to do with a table wrapping the pagination component but rather a bordered container we might call this something more informational i.e. `.container-bordered` and maybe make it a ` border: 2px solid whitesmoke;` if the extra border bottom isn't a problem.

Reply:
`<Container className="border border-light border-2 border-top-0"> ... </Container>`

Reply:
I can change the name of the class.
Using bootstrap border classes if feasible, but not in this PR.
We need to clean/uniformize the Admin panel tables, as every table has been made a different way.

Reply:
That's good,
For the divergence of all admin panel tabs, that is due to the decision we took by reducing complexity and duplicating the panels avoiding the use of `<Outlet />`.
This might help #3590

Reply:
Exactly, I have no problem with the duplication.

They should be uniformized, not DRY-ed.

It seems like every table has been built differently, and now behaves differently.

Reply:
I think if it's gonna take sometime then this may be better to be done at the end of development otherwise after uniformizing them now they may diverge again in the upcoming weeks,

Reply:
Good point.